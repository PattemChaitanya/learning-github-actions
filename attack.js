const attack = (name, damage, isCritical) => `${name} dealt ${isCritical ? damage * 2 : damage} damage!`;

module.exports = attack;
