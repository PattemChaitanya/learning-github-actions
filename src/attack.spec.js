import attack from './attack';

test('should attack return', () => {
  const result = attack('sony', 23);
  // expect(result).toBe('sony dealt 23 damage!');
  expect(result).toMatchSnapshot();
});
