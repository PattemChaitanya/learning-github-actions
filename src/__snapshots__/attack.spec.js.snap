// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should attack return 1`] = `"sony dealt 23 damage!"`;
